src/renderers/common/extras/PMREMGenerator.js

```diff
@@ -135,13 +135,15 @@ class PMREMGenerator {
 	 * @param {Number} [sigma=0] - The blur radius in radians.
 	 * @param {Number} [near=0.1] - The near plane distance.
 	 * @param {Number} [far=100] - The far plane distance.
+	 * @param {Number} [size=256] - The size (resolution) of the cube texture,
+	 * @param {Number} [position = new Vector3( 0, 0, 0 )] - The cubeCamera position
 	 * @param {RenderTarget?} [renderTarget=null] - The render target to use.
 	 * @return {RenderTarget} The resulting PMREM.
 	 * @see fromSceneAsync
 	 */
-	fromScene( scene, sigma = 0, near = 0.1, far = 100, renderTarget = null ) {
+	fromScene( { scene, sigma = 0, near = 0.1, far = 100, size = 256, position = new Vector3( 0, 0, 0 ), renderTarget = null } ) {
```

Comment:
Can you please revert this change? I understand an object as a parameter is more flexible but unfortunately such a change breaks user code.

Changing the signature from a parameter list to an object needs to be discussed/implemented in a separate issue. Maybe it makes sense to also update the generator for `WebGLRenderer`.